app/src/main/java/app/grapheneos/camera/ui/activities/MoreSettings.kt

```diff
@@ -96,6 +96,14 @@ open class MoreSettings : AppCompatActivity(), TextView.OnEditorActionListener {
                 sIAPToggle.isChecked
         }
 
+        val sIVPToggle = binding.saveVideoAsPreviewToggle
```

Comment:
What does `IVP` stand for?

Reply:
Ah my bad, it should have been sVAPToggle implying "Save Video As Previewed Toggle". I have resolved the latest changes to the branch